packages/core/admin/admin/src/content-manager/components/DynamicZone/components/ComponentPicker/Category/ComponentCard/index.js

```diff
@@ -62,7 +62,7 @@ function ComponentCard({ componentUid, intlLabel, icon, onClick }) {
   return (
     <button type="button" onClick={handleClick}>
       <ComponentBox borderRadius="borderRadius">
-        <Stack size={1} style={{ justifyContent: 'center', alignItems: 'center' }}>
+        <Stack spacing={1} style={{ justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
There is no `justifyContent` nor `alignItems` props in the Box component.

Reply:
Yes but the Flex has I guess we could wrap everything in the Flex one instead of overriding the styles.

Will do this in another PR though.

Can you just add a comment so we remove this inline style?